CHANGELOG.md

```diff
@@ -23,12 +23,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    the non-coalesced case.
  - decoder API: new function `JxlDecoderGetExtraChannelBlendInfo` to get
    the blending information for extra channels in the non-coalesced case.
+ - encoder API: added ability to set several encoder options to frames using
+   `JxlEncoderSetFrameSetting`
```

Comment:
There's also `JxlEncoderSetFrameInfo`.

Maybe those two names are a bit confusing. One is to change an encode option, the other is to update frame header info.

Perhaps the following names are clearer:

- `JxlEncoderFrameSettingsSetEncodeOption`  (instead of `JxlEncoderSetFrameSetting`)
- `JxlEncoderFrameSettingsSetHeaderInfo` (instead of `JxlEncoderSetFrameInfo`)

(or JxlEncoderSetFrameSettingsOption / JxlEncoderSetFrameSettingsInfo)

In any case I would consistently use plural `FrameSettings` even if the function is only used to change one setting.

Reply:
Makes sense, I chose the following names, with the word "Set" later because especially for the frame name the early "Set" was confusing:

 - JxlEncoderFrameSettingsSetOption
 - JxlEncoderFrameSettingsSetInfo
 - JxlEncoderFrameSettingsSetName